build-plugin/action.yml

```
@@ -34,22 +34,26 @@ inputs:
     type: choice
     description: "Backend target for the plugin backend build"
     options:
-        - build:backend
-        - build:darwin
-        - build:darwinARM64
-        - build:debug
-        - build:debugDarwinAMD64
-        - build:debugDarwinARM64
-        - build:debugLinuxAMD64
-        - build:debugLinuxARM64
-        - build:debugWindowsAMD64
-        - build:linux
-        - build:linuxARM
-        - build:linuxARM64
-        - build:windows
-        - buildAll
+      - build:backend
+      - build:darwin
+      - build:darwinARM64
+      - build:debug
+      - build:debugDarwinAMD64
+      - build:debugDarwinARM64
+      - build:debugLinuxAMD64
+      - build:debugLinuxARM64
+      - build:debugWindowsAMD64
+      - build:linux
+      - build:linuxARM
+      - build:linuxARM64
+      - build:windows
+      - buildAll
     required: false
     default: "buildAll"
+  attestation:
+    description: "Create a verifiable attestation for the plugin using Github OIDC. Requires id-token: write and attestations: writes permissions"
+    required: false
+    default: "false"
```

Can we make this a boolean?

```suggestion
  attestation:
    description: "Create a verifiable attestation for the plugin using Github OIDC. Requires id-token: write and attestations: writes permissions"
    required: false
    type: boolean
    default: false
```